docs/geedocs/answer/7160000.html

```diff
@@ -0,0 +1,204 @@
+        </tr>
+        <tr>
+          <td>326</td>
+          <td>Libraries may be loaded from the wrong directory</td>
+          <td>Delete any library versions that should not be loaded or use LD_LIBRARY_PATH to load libraries from <code>/opt/google/lib</code>.</td>
+        </tr>
+        <tr>
+          <td>333</td>
+          <td>Portable Server is not supported on MacOS</td>
+          <td>Building and running Portable Server on MacOS should be possible with minimal changes.</td>
+        </tr>
+        <tr>
+          <td>335, 359</td>
+          <td>If there is an error while saving a resource, the resource cannot be saved again even if the error is resolved</td>
+          <td>Close the resource form and open it again to make the save option available again.</td>
+        </tr>
+        <tr>
+          <td>342</td>
+          <td>Fusion crashes when opening an unsupported file type</td>
+          <td>Do not open unsupported file types.</td>
+        </tr>
+      </tbody>
+    </table>
+  <h5>
+    <p id="resolved_issues_5.2.0">Resolved Issues</p>
+  </h5>
+  <div>
+    <table class="nice-table">
+      <tbody>
+        <tr>
+          <th>Number</th>
+          <th class="narrow">Description</th>
+          <th>Resolution</th>
+        </tr>
+        <tr>
+          <td>(none)</td>
+          <td>Error when fusing a mosaic in GDAL 2.x</td>
+          <td>Implemented <code>IReadBlock()</code> API for <code>khVRRasterBand</code></td>
+        </tr>
+        <tr>
+          <td>16</td>
+          <td>gefusionuser must have write access to vector files</td>
+          <td>Removed the requirement for write access</td>
+        </tr>
+        <tr>
+          <td>26</td>
+          <td>Fusion segmentation faults when trying to push and the server is not available</td>
+          <td>Pass the correct data type to cURL function</td>
+        </tr>
+        <tr>
+          <td>167</td>
+          <td>If save fails because the Fusion server is not running, the user cannot save again even if the problem is fixed</td>
+          <td>Updated state management in resource creation form</td>
+        </tr>
+        <tr>
+          <td>179</td>
+          <td>Update date parsing to fix default date handling</td>
+          <td>Added code to handle default dates in Fusion tools</td>
+        </tr>
+        <tr>
+          <td>239</td>
+          <td><code>cachedreadaccessor_unittest</code> fails on <code>free()</code> call on Ubuntu 16.04</td>
+          <td>Increased the size of a buffer that was overflowing</td>
+        </tr>
+        <tr>
+          <td>243</td>
+          <td>Fix broken line in cutter script</td>
+          <td>Fixed the relevant line</td>
+        </tr>
+      </tbody>
+    </table>
+  <div class="footer">
+    <p class="BackToTop"><a href="#top_of_file">Back to top</a>
+    
+    <hr />
+    </p>
+    <p class="copyright">&copy;2015 Google</p>
```

Comment:
Tell me what to put there and I'll change it.  It might be easier to just write a ticket to change all the copyrights.